lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1022,6 +1022,14 @@ def _create_deployment_parameters(
         ):
             arm_parameters.use_availability_sets = True
 
+        if environment.platform and (
```

Comment:
Use `assert environment.platform`, so the logic is clearer. It's not a condition of the if, it's to pass static type checking.